File: Day 1/1 Find Characters in String/solution.cpp
#include <iostream>
#include <string>
#include <vector>

int findCharcterInString(std::string word, char character) {
  for (int i = 0; i < word.length(); i++) {
    if (word[i] == character) {
      return i;
    }
  }
  return -1;
}

std::vector<int> findAllCharcterIndexesInString(std::string word, char character) {
  std::vector<int> indexes;
  for (int i = 0; i < word.length(); i++) {
    if (word[i] == character) {
      indexes.push_back(i);
    }
  }
  return indexes;
}

File: Day 1/1 Find Characters in String/solution.js
function findCharcterInString(word, character) {
    for (let i = 0; i < word.length; i++) {
      if (word.charAt(i) == character) {
        return i;
      }
    }
    return -1;
  }
  
  function findAllCharcterIndexesInString(word, character) {
    const indexes = [];
    for (let i = 0; i < word.length; i++) {
      if (word.charAt(i) == character) {
        indexes.push(i);
      }
    }
    return indexes;
  }
  